Add public Vale configuration (#287)
grafana/writers-toolkit: docs/sources/write/front-matter/index.md

The `date` front matter also impacts menu ordering.
Pages with more recent dates are lower in the menu.

### `description` (required)

> Use to provide the short description of the topic to search engines, including the search engine used in the Grafana documentation site. The description is also displayed on social media, such as Twitter, to provide a clue to users about the page contents.

Reviewer: change name to X?

Author: Lol! I'm not sure Twitter has lost meaning yet. At some point this change might be needed but I don't think it's necessary yet.

If it's too long, it is harmlessly truncated on social media.
Use double quotes (`"`) to surround the title. Do not use smart quotes.

### `draft`

When set to `true`, this option prevents Hugo from rendering the content.

> Use the command line flag `--buildDrafts` to generate content marked as `draft: true`.

Reviewer: ```suggestion
Use the command-line flag `--buildDrafts` to generate content marked as `draft: true`.
```

      - cloud
```

### `menuTitle`

> Use to specify a different heading in the sidebar navigation than the `title` element, for example if you want to abbreviate the topic heading in the table of contents.

Reviewer: ```suggestion
Use to specify a different heading in the sidebar navigation than the `title` element; for example, if you want to abbreviate the topic heading in the table of contents.
```

### `title` (required)

Hugo uses the `title` to generate the sidebar table of contents if there is no `menuTitle` specified in the front matter. If the `doc-validator` linter has been implemented on your repository, your topic heading must exactly match the title in the metadata.

> The `title` becomes the document title element in the HTML. Often browsers display this in the tab for the page.

Reviewer: ```suggestion
The `title` becomes the document title element in the HTML. Often, browsers display this in the tab for the page.
```

Therefore, with the alias `./alerting/unified-alerting/` in the source file for the page `/docs/grafana/latest/alerting/`, Hugo will create a redirect page at `/docs/grafana/latest/alerting/unified-alerting/`.

#### Other projects

Include an `aliases` entry for the current URL path.

> Adding an `aliases` entry makes it safer to move content around as the redirect from old to new page location is already in place.

Reviewer: ```suggestion
Addi an `aliases` entry to make it safer to move content around, as the redirect from old to new page location is already in place.
```
